src/uids.js

```diff
@@ -1,22 +1,42 @@
-const octetStream = 'application/octet-stream';
```

Comment:
@wayfarer3130  these variable declaration is something I would like to discuss with you.